projects/common-components/src/lib/talks/talk-item/talk-item.component.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { ComponentFixture, TestBed } from "@angular/core/testing";
+import { TalkItemComponent } from "@jsmu/common-components";
+import { Result } from "@jsmu/core";
+
+describe("TalkItemComponent", () => {
+	let component: TalkItemComponent;
+	let fixture: ComponentFixture<TalkItemComponent>;
+	beforeEach(async () => {
+		await TestBed.configureTestingModule({
+			declarations: [TalkItemComponent]
+		}).compileComponents();
+
+		fixture = TestBed.createComponent(TalkItemComponent);
+		component = fixture.componentInstance;
+		component.talk = {
```

Comment:
please make in assets test folder with ready mock object and export from there everywhere. It will save space in the code =)

Reply:
done